coveralls.json

```diff
@@ -3,7 +3,6 @@
     "minimum_coverage": 30
   },
   "skip_files": [
-    "elixir_boilerplate/release/migration.ex",
```

Comment:
Were we supposed to ignore [this file](https://github.com/mirego/elixir-boilerplate/blob/master/lib/elixir_boilerplate/release_tasks.ex) instead?

Reply:
I’m not familiar with the history of this project so I can’t tell if it has just been renamed. Should I add it to the ignore list?

Reply:
I feel like it is something we might want to test, no?

Reply:
Technically, this file contains the migration process when deploying an OTP release, which is used by this script: 

https://github.com/mirego/elixir-boilerplate/blob/master/rel/commands/migrate.sh

I feel like it must be ignored, indeed.

Reply:
And yes, it should be the file proposed by @remiprev that is added there as `skip_files`.

Reply:
I’ve added `mix/tasks/erlang.check_version.ex` as well.